spec/workflow/reference_path_spec.rb

```diff
@@ -50,19 +50,27 @@
 
   describe "#set" do
     let(:payload) { "$" }
-    let(:input) { {} }
+    let(:input) { { "old" => "key"} }
```

Comment:
this was an empty hash before, so the tests did not show the distinction here.
Adding a before value caused the code to show the nuance I am highlighting